### ISSUE TITLE
setLocation with custom fieldNames

### ISSUE THREAD

**Issue by Seeyko** [imperiumlabs/GeoFirestore-Android#40]
Hello,

I have a firestore database with already existing GeoPoint under the field : `"gpsPoint"`
When i set a new location with GeoFirestore i use this already existing `gpsPoint` field in `setLocation ` method. But it also add the `"l"` field with a GeoPoint wich is a duplication of my `"gpsPoint"`.

**Comment by Supercaly** [comment]
Yes, currently the library creates his own fields "l" and "h" and use them as indexes for queries; there is no way to make the library use custom fields, but if you want you can implement this feature and make a pull request, it will be appreciated.

**Comment by betegon** [comment]
Any update on this @Seeyko ?
